### DISCUSSION TITLE
FYI - ZoeDepth does not work with SDP attention flags in pytorch stable 2.0.1

### DISCUSSION THREAD

**Discussion by hithereai** [Mikubill/sd-webui-controlnet#1224] [General]
No attention flags on a1111 with pt 2.0.1:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/121192995/0a83b782-af7e-48c8-b1e5-f8f6e3497f9a)

With SDP attention flags:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/121192995/248c10de-c7fe-480b-866f-d0b8d8bf00cb)

An issue needs to be opened in [the Zoe](https://github.com/isl-org/ZoeDepth) but I'm not sure how to actually report it, since it only happens with SDP flags.

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/121192995/c3ea4977-8d64-4e70-8735-5f9cbbe41db4)

**Comment by lllyasviel** [comment]
why zoe read a1111 flag?

**Reply by hithereai** [reply]
That's the thing - it does not. And it works perfectly on pytorch 2.0, so... I wonder. Getting the same issues both on your Zoe implementation, and on mine (in deforum). 

Also works perfectly with pt 2.0.1, just without the SDP attention flags.

**Comment by kabachuha** [comment]
## Midas beit large 512 doesn't work with SDP added and turned on as well

Torch 2.01 without SDP:

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/14872007/0cf2eb33-cc18-4537-8305-66296817e144)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/14872007/1cbf7514-4f6e-4c3f-a005-90739c1ddac2)

Torch 2.01 with SDP:

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/14872007/e4471c28-aab0-4765-a079-399530b8e118)

Pics by @hithereai
